RecoLocalTracker/Phase2TrackerRecHits/python/Phase2StripCPEESProducer_cfi.py

```diff
@@ -2,5 +2,8 @@
 
 phase2StripCPEESProducer = cms.ESProducer("Phase2StripCPEESProducer",
   ComponentType = cms.string('Phase2StripCPE'),
-  parameters    = cms.PSet()
+  parameters    = cms.PSet(
+    LorentzAngle_DB = cms.bool(False),			
+    TanLorentzAnglePerTesla = cms.double(0.07)
```

Comment:
shouldn't it be 0.07 for pixel and 0.106 for strips ? it is at least how it is defined in the digitzer  https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py